lib/classes/organisations_user_builder.rb

```diff
@@ -0,0 +1,33 @@
+class OrganisationsUserBuilder
+
+  def initialize(organisations_user)
```

Comment:
@abulsayyad123 what data is inside `organisations_user`? I think I'm missing something here and it would be helpful to get a sense of it. I was expecting something like `{organisation_id: 1, mobile: "+85212345678", name: "Bob Jones", email: "bob@example.com"}`

Then the mobile would be the key, whether the user exists or not? Perhaps you could clarify for me what data the ember controller posts to the API in both cases. 1) when the user already exists, and 2) when they don't exist. Thanks